### PR TITLE
some gtk3 preparation

### PR DESCRIPTION
**Pull request by necessarily-equal**

### CODE REVIEW

File: src/flt-player.c [hunk 1]
```diff
@@ -1178,7 +1178,7 @@ static void player_filter_load_patterns (void) {
 	scanner = g_scanner_new (&patterns_scanner_config);
 
 	for (i = TOKEN_STRING; i <= TOKEN_REGEXP; i++) {
-		g_scanner_add_symbol (scanner, mode_symbols[i - TOKEN_STRING],
+		g_scanner_scope_add_symbol (scanner, 0, mode_symbols[i - TOKEN_STRING],
```
Comment (illwieckz) [inline src/flt-player.c:1175]: while you're at it, can you remove the line split (cleaning-up the style at the same time)?

File: src/scripts.c [hunk 2]
```diff
@@ -606,7 +606,7 @@ GtkWidget *scripts_config_page () {
 	button = gtk_button_new_with_label(_("Install..."));
 	gtk_box_pack_start (GTK_BOX (vbox), button, FALSE, FALSE, 0);
 
-	g_signal_connect (GTK_OBJECT (button),
+	g_signal_connect (G_OBJECT (button),
```
Comment (illwieckz) [inline src/scripts.c:609]: same, can you fix style of those edited lines at the same time?

Comment (illwieckz) [reply src/scripts.c:609]: well, github was hidding me all the changes, don't worry about this.

File: src/flt-player.c (hunk 1 above)
Comment (illwieckz) [reply src/flt-player.c:1175]: At least this file has only one change so that can be done there. otherwise don't worry.

Comment (necessarily-equal) [reply src/flt-player.c]: updated

**Review by illwieckz — APPROVED** [review]

**Comment by illwieckz** [comment]
Thank you very much!

**Pull request merged**